banyand/measure/write_liaison.go

```diff
@@ -117,8 +117,18 @@ func (w *writeQueueCallback) Rev(ctx context.Context, message bus.Message) (resp
 		g := groups[i]
 		for j := range g.tables {
 			es := g.tables[j]
+			// Marshal series metadata for persistence in part folder
+			var seriesMetadataBytes []byte
+			if len(es.metadataDocs) > 0 {
+				var marshalErr error
+				seriesMetadataBytes, marshalErr = es.metadataDocs.Marshal()
```

Comment:
Remove "es.metadataDocs.Marshal()" from `sendDocuments` to avoid duplicated marshaling.